### ISSUE TITLE
How to represent a table of data

### ISSUE THREAD

**Issue by vladejs** [vazco/uniforms#153]
Hello @radekmie, i have to say that you have created a great package, congratulations. I am using it extensively and I am facing a use case when I need to get data where the schema is a table, for example, services provided in all week days:
```
Service     Sunday  Tuesday ...
service 1       x             
service 2               x
```

The above table means that service1 was provided on Sunday but not on Tuesday and service2 was provided on tuesday but not on sunday.

Do you have any recommendation on addressing this data structure? I think that a blackbox Object with a custom component should do the trick, but I would like to hear your thoughts.

Thanks in advance.

**Comment by radekmie** [comment]
If your data is finite and you want to have a fine grained control, then create a very detailed schema. Blackbox objects are also applicable here, but in my opinion those should be used only for _not-schemable_ data.

About the component - it is always better to have a tailored one for a very specific data. In this very case, a component for this whole object is a good idea.

**Comment by radekmie** [comment]
Also, if the list of services has variable length, a custom `ListField` is also a good idea - I've also used one to represent user accesses, which seems to be a similar case.

**Comment by vladejs** [comment]
The list of services is fixed in length. I created a very detailed schema, but I don't know to take it into a table. 

I ended up using a custom component which uses the exceptional `react-bootstrap-table` package, which allows me to edit table cells just by clicking on them. I call `onChange` function on every cell edit and it does the trick.

For reference, this is how I wrote the schema:
```coffeescript
hoursOfService:
    type: [Object]
    optional: true
    blackbox: true
    uniforms:
      component: TableFieldComponent
      columns: columnsHoursOfService
      data: dataHoursOfService
```
